frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -195,43 +192,45 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                 loadRecordingMetaFailure: () => true,
             },
         ],
+        snapshotsLoaded: [
+            false as boolean,
+            {
+                loadRecordingSnapshotsSuccess: () => true,
+                loadRecordingSnapshotsFailure: () => true,
```

Comment:
Should this be `false`?

Reply:
`loadSnapshotsTried` `loadSnapshotsStarted` ??